packages/frameworks/src/frameworks.ts

```diff
@@ -546,6 +546,47 @@ export const frameworks = [
       },
     ],
   },
+  {
+    name: 'SolidStart',
+    slug: 'solidstart',
+    demo: 'https://solidstart.examples.vercel.com',
+    logo: 'https://raw.githubusercontent.com/vercel/vercel/main/packages/frameworks/logos/solid.svg',
+    tagline: 'Simple and performant reactivity for building user interfaces.',
+    description: 'A Solid app, created with SolidStart.',
+    website: 'https://solidjs.com',
+    envPrefix: 'VITE_',
+    detectors: {
+      every: [
+        {
+          path: 'package.json',
+          matchContent:
+            '"(dev)?(d|D)ependencies":\\s*{[^}]*"solid-js":\\s*".+?"[^}]*}',
+        },
+        {
+          path: 'package.json',
+          matchContent:
+            '"(dev)?(d|D)ependencies":\\s*{[^}]*"solid-start":\\s*".+?"[^}]*}',
+        },
+      ],
+    },
+    settings: {
+      installCommand: {
+        placeholder: '`yarn install` or `npm install`',
+      },
+      buildCommand: {
+        placeholder: '`npm run build` or `solid-start build`',
+        value: 'solid-start build',
+      },
+      devCommand: {
+        value: 'solid-start dev',
+      },
+      outputDirectory: {
+        value: '.output',
```

Comment:
I think the output directory here must be the static output, because I tried deploying and this doesn't work:

https://solidstart-1ovuv4mzd-now-examples.vercel.app/

cc @AndyBitz @TooTallNate

Reply:
Interesting, it seems like we need a way to support adding a framework to this list, that supports the File System API by default, unless this directory is the static directory.

Reply:
```suggestion
        value: '.output/static',
```

I confirmed, this should be static only

Reply:
I tried deploying the example code here with Output Directory set to `.output/static` but that doesn't work because there is no index.html there either.

Maybe we need an indicator here to say that this framework uses the File System API?

Reply:
Any updates on this? Anything I can do to help?

Reply:
We'll make sure that `solidstart` is always enabled to the File System API beta.

Also, this PR will reduce the confusion for new frameworks: #7124

Reply:
We're going to keep `outputDirectory` here pointing at the value of the static output, not the `.output`